src/components/graph/util/use-modification-label-computer.jsx

```diff
@@ -6,7 +6,7 @@
  */
```

Comment:
File to remove ? This code is available in commons-ui

Reply:
I changed the links but forgot to delete it !  Thank you.